library/Leads/Export.php

```diff
@@ -263,10 +277,14 @@ protected function getExportData($objConfig, $arrIds=null)
             if ($objConfig->export == 'all') {
                 \System::loadLanguageFile('tl_lead_export');
 
+                $arrHeader[] = $GLOBALS['TL_LANG']['tl_lead_export']['_pid']; // insert ID in export-file (added by CO/BODA - Ronald Boda)
```

Comment:
this translation does not seem to exist?

Reply:
pushed changed files and additional translation files